chess/syzygy.py

```diff
@@ -108,187 +108,187 @@ def flipdiag(square: chess.Square) -> chess.Square:
 FILE_TO_FILE = [0, 1, 2, 3, 3, 2, 1, 0]
 
 KK_IDX = [[
-     -1,  -1,  -1,   0,   1,   2,   3,   4,
-     -1,  -1,  -1,   5,   6,   7,   8,   9,
-     10,  11,  12,  13,  14,  15,  16,  17,
-     18,  19,  20,  21,  22,  23,  24,  25,
-     26,  27,  28,  29,  30,  31,  32,  33,
-     34,  35,  36,  37,  38,  39,  40,  41,
-     42,  43,  44,  45,  46,  47,  48,  49,
-     50,  51,  52,  53,  54,  55,  56,  57,
+    -1,  -1,  -1,   0,   1,   2,   3,   4,
+    -1,  -1,  -1,   5,   6,   7,   8,   9,
+    10,  11,  12,  13,  14,  15,  16,  17,
+    18,  19,  20,  21,  22,  23,  24,  25,
+    26,  27,  28,  29,  30,  31,  32,  33,
+    34,  35,  36,  37,  38,  39,  40,  41,
+    42,  43,  44,  45,  46,  47,  48,  49,
+    50,  51,  52,  53,  54,  55,  56,  57,
 ], [
-     58,  -1,  -1,  -1,  59,  60,  61,  62,
-     63,  -1,  -1,  -1,  64,  65,  66,  67,
-     68,  69,  70,  71,  72,  73,  74,  75,
-     76,  77,  78,  79,  80,  81,  82,  83,
-     84,  85,  86,  87,  88,  89,  90,  91,
-     92,  93,  94,  95,  96,  97,  98,  99,
-    100, 101, 102, 103, 104, 105, 106, 107,
-    108, 109, 110, 111, 112, 113, 114, 115,
+    58,  -1,  -1,  -1,  59,  60,  61,  62,
+    63,  -1,  -1,  -1,  64,  65,  66,  67,
+    68,  69,  70,  71,  72,  73,  74,  75,
+    76,  77,  78,  79,  80,  81,  82,  83,
+    84,  85,  86,  87,  88,  89,  90,  91,
+    92,  93,  94,  95,  96,  97,  98,  99,
+   100, 101, 102, 103, 104, 105, 106, 107,
```

Comment:
Lines starting with a 3 digit number are now underindented (only 3 spaces at the start).

Reply:
Then leave these matrices that start with a 3-digit number as they are.